src/ucs/datastruct/array.inl

```diff
@@ -8,7 +8,7 @@
 #define UCS_ARRAY_INL_
 
 #include "array.h"
-
+#include <ucs/type/status.h>
```

Comment:
1. space line before and after
2. add it to array.h because it uses ucs_status_t

Reply:
array.h has status only inside macros, so it's not really needed threre.

Reply:
@shasson5 every file which `#include`s `array.h` must have `ucs_status_t`, therefore better it is included from there.